pythonx/UltiSnips/snippet/definition/_base.py

```diff
@@ -78,6 +81,32 @@ def _re_match(self, trigger):
             return match
         return False
 
+    def _context_match(self):
+        current = vim.current
+        # skip on empty buffer
+        if len(current.buffer) == 1 and current.buffer[0] == "":
```

Comment:
This requires some more tests. Also I do not think this is python3 compatible - I need to add that to the travis tests :/